plugins/autogems.cpp

```diff
@@ -146,6 +190,9 @@ void create_jobs() {
             continue;
         }
 
+        if (workshop->isForbidden())
+            continue;
+
```

Comment:
This appears to be an entirely unrelated change.  Meanwhile, how often do buildings get forbidden on purpose, instead of by accident?  How likely is this to surprise people?